src/datepicker/datepicker-month-view.scss

```diff
@@ -20,6 +20,8 @@ ngb-datepicker-month-view {
   &-weekdays {
     border-bottom: 1px solid rgba(0, 0, 0, 0.125);
     border-radius: 0;
+    background-color: #f8f9fa;
```

Comment:
Removing `bg-light` class but you are also introducing a hard coded background color. Why not set on parent background color and in custom css it can be overridden?

Reply:
`.ngb-datepicker-month-view-weekdays` is a custom-class publicly available.
You can override the background-color by defining your own. Isn't it enough ?

Reply:
I can but I have to override for each including header, week, weekday, day using `!important`